### ISSUE TITLE
Question: Ignore HasPrefix and HasSuffix case

### ISSUE THREAD

**Issue by dwaynelavon** [prisma/prisma-client-go#128]
Is it possible to ignore the case on a FindMany query with HasPrefix or HasSuffix?

Something like:
`client.User.FindMany(User.Name.HasPrefix("bob", true))` where the true argument denotes whether or not to ignore case.

Or: 
`client.User.FindMany(User.Name.ToLower().HasPrefix("bob"))` so that a user named Bob will still return in the results

**Comment by steebchen** [comment]
Unfortunately this is currently not possible. The related feature request for the Prisma query engine can be found on the Prisma repo: https://github.com/prisma/prisma-client-js/issues/690

**Comment by darmie** [comment]
When will this be supported, the main Prisma doc has a `mode` feature to determine case sensitivity, 
```js
const users = await prisma.user.findMany({
  where: {
    email: {
      endsWith: "prisma.io",
      mode: "insensitive", // Default value: default
    },
  },
});
```

How do I do this in Go?

**Comment by steebchen** [comment]
Not possible in the Go client yet, I'll bring this up in the next planning meeting.

**Comment by njfix6** [comment]
@steebchen do you know when this feature will be released for golang?

**Comment by steebchen** [comment]
No, but assuming you want to see this feature, I can see if we have capacity to work on this soon in our next planning meeting.
